clang/utils/TableGen/SveEmitter.cpp

```diff
@@ -87,6 +87,10 @@ class SVEType {
   bool isDefault() const { return DefaultType; }
   bool isFloat() const { return Float && !BFloat; }
   bool isBFloat() const { return BFloat && !Float; }
```

Comment:
To be consistent I think these need updating to include `&& !MFloat`?

There's also an "isFloatingPoint()" method that probably needs `|| MFloat`.